neuron_explainer/models/transformer.py

```diff
@@ -722,21 +721,15 @@ def save_checkpoint(
 
         pieces_path = osp.join(path, "model_pieces")
         for k, v in self.state_dict().items():
-            with bf.BlobFile(osp.join(pieces_path, f"{k}.pt"), "wb") as f:
+            with CustomFileHandler(osp.join(pieces_path, f"{k}.pt"), "wb") as f:
                 torch.save(v, f)
 
     def load_state_from_checkpoint(
         self, path: str, device: Device | None = None, dtype: torch.dtype | None = None
     ):
         pieces_path = osp.join(path, "model_pieces")
-        piece_files = list(bf.listdir(pieces_path))
```

Comment:
again, couldn't find a good way to list files in a directory when you only have the https link